docs/quick-start-guide.md

```diff
@@ -52,6 +52,10 @@ If you would like to avoid doing the above steps manually, we have prepared a
 project directory, run a `vagrant up`, and then a `vagrant status`. From there,
 you can `vagrant ssh` into the `mgmt` machine. The MOTD will explain the rest.
 
+### Using Docker
```

Comment:
Should be `## Docker` instead. Note the existing headings.

Reply:
done